README.md

```diff
@@ -103,10 +105,7 @@ The Lob constructor accepts an `options` object which may contain one or more of
 
 We've provided various examples for you to try out [here](https://github.com/lob/lob-node/tree/master/examples).
```

Comment:
Should this link to the directory or the README? Or is the idea that if you're linked to a directory with a readme, you better read it?